recipe/meta.yaml

```
@@ -24,21 +25,16 @@ requirements:
     - {{ compiler('cxx') }}
     - file  # [arm64]
   host:
-    - zlib
     - bzip2
-    - zstd
+    - curl
     - lz4-c
+    - google-cloud-cpp # [gcs == 'gcs_enabled']
```

```suggestion
    - google-cloud-cpp  # [gcs == 'gcs_enabled']
```
This will make the linter happy.